package/MDAnalysis/topology/TPRParser.py

```diff
@@ -171,12 +171,24 @@ class TPRParser(TopologyReaderBase):
     """
     format = 'TPR'
 
-    def parse(self, **kwargs):
+    def parse(self, tpr_resid_from_one=False, **kwargs):
         """Parse a Gromacs TPR file into a MDAnalysis internal topology structure.
 
+        Parameters
+        ----------
+        tpr_resid_from_one: bool (optional)
+            Toggle whether to index resids from 1 or 0 from TPR files.
+            TPR files index resids from 0 by default, even though GRO and ITP
+            files index from 1.
+
         Returns
         -------
         structure : dict
+
+
+        .. versionchanged:: 1.0.2
+            Added the ``tpr_resid_from_one`` keyword to control if
+            resids are indexed from 0 or 1. Default ``False``.
```

Comment:
Becomes 1.1.0 ...

Reply:
I thought we were going to leave 1.x at the 1.0.x states? It's becoming increasingly difficult to deal with Python 2.7, as evidenced by the recent issues with CI. What about putting the deprecation in 2.0 and then making the final change in a later release?

Tagging any interested @MDAnalysis/coredevs.